manim/renderer/webgl_renderer.py

```diff
@@ -59,6 +60,7 @@ def get_frame(self):
         pass
 
 
+@dataclass
```

Comment:
What's the reasoning behind using a dataclass here?

Reply:
Flake was telling me to, so I started fixing that ignore. I found it was not worth it and just ignored it. I can remove the dataclass if you want, though this is the exact case dataclass is designed for.

Reply:
It doesn't feel related to this PR (and isn't implemented properly as it's overwriting the `__init__` a dataclass would generate) so I think it'd be better to remove it.

Reply:
ok, I'll get on that